src/aklite_client_ext.cc

```diff
@@ -276,3 +277,51 @@ bool AkliteClientExt::RebootIfRequired() {
 
   return false;
 }
+
+InstallResult AkliteClientExt::Rollback(const LocalUpdateSource* local_update_source, int version,
+                                        const std::string& target_name) {
+  auto pending_target = Target::toTufTarget(client_->getPendingTarget());
```

Comment:
I suppose we can replace it with `GetPendingTarget()` (the base class implements it).